internal/seclang/directives.go

```diff
@@ -607,25 +618,37 @@ func directiveSecAuditLogRelevantStatus(options *DirectiveOptions) error {
 // Syntax: SecAuditLogParts [PARTLETTERS]
 // Default: ABCFHZ Note
 // ---
-// The format of the audit log format is documented in detail in the Audit Log Data Format Documentation.
+// The format of the audit log format is documented in detail in the Audit Log Data
+// Format Documentation.
+//
 // Example:
-// ```
+// ```apache
 // SecAuditLogParts ABCFHZ
 // ```
 //
 // Available audit log parts:
 //
 // A: Audit log header (mandatory).
 // B: Request headers.
-// C: Request body (present only if the request body exists and ModSecurity is configured to intercept it. This would require SecRequestBodyAccess to be set to on).
+// C: Request body (present only if the request body exists and ModSecurity is configured
+// to intercept it. This would require SecRequestBodyAccess to be set to on).
 // D: Reserved for intermediary response headers; not implemented yet.
-// E: Intermediary response body (present only if ModSecurity is configured to intercept response bodies, and if the audit log engine is configured to record it. Intercepting response bodies requires SecResponseBodyAccess to be enabled). Intermediary response body is the same as the actual response body unless ModSecurity intercepts the intermediary response body, in which case the actual response body will contain the error message (either the Apache default error message, or the ErrorDocument page).
-// F: Final response headers (excluding the Date and Server headers, which are always added by Apache in the late stage of content delivery).
+// E: Intermediary response body (present only if ModSecurity is configured to intercept
+// response bodies, and if the audit log engine is configured to record it. Intercepting
+// response bodies requires SecResponseBodyAccess to be enabled). Intermediary response
+// body is the same as the actual response body unless ModSecurity intercepts the intermediary
+// response body, in which case the actual response body will contain the error message (either
+// the Apache default error message, or the ErrorDocument page).
```

Comment:
```suggestion
// response body, in which case the actual response body will contain the error message.
```
nit: Should we be more server agnostic avoiding a direct reference to Apache (or adding it as a E.g., if it really is the case)?